--- a/src/riakc_pb_socket.erl
+++ b/src/riakc_pb_socket.erl
@@ -27,15 +27,15 @@
 
 -module(riakc_pb_socket).
 -include_lib("kernel/include/inet.hrl").
--include_lib("riak_pb/include/riak_pb.hrl").
--include_lib("riak_pb/include/riak_kv_pb.hrl").
--include_lib("riak_pb/include/riak_pb_kv_codec.hrl").
--include_lib("riak_pb/include/riak_ts_pb.hrl").
--include_lib("riak_pb/include/riak_search_pb.hrl").
--include_lib("riak_pb/include/riak_yokozuna_pb.hrl").
--include_lib("riak_pb/include/riak_dt_pb.hrl").
--include_lib("riak_pb/include/riak_ts_pb.hrl").
--include_lib("riak_pb/include/riak_ts_ttb.hrl").
+-include_lib("riak_protobuf/include/riak_protobuf.hrl").
+-include_lib("riak_protobuf/include/riak_kv_pb.hrl").
+-include_lib("riak_protobuf/include/riak_protobuf_kv_codec.hrl").
+-include_lib("riak_protobuf/include/riak_ts_pb.hrl").
+-include_lib("riak_protobuf/include/riak_search_pb.hrl").
+-include_lib("riak_protobuf/include/riak_yokozuna_pb.hrl").
+-include_lib("riak_protobuf/include/riak_dt_pb.hrl").
+-include_lib("riak_protobuf/include/riak_ts_pb.hrl").
+-include_lib("riak_protobuf/include/riak_ts_ttb.hrl").
 -include("riakc.hrl").
 -behaviour(gen_server).
 
--- a/src/riakc_obj.erl
+++ b/src/riakc_obj.erl
@@ -72,7 +72,7 @@
 %% Internal library use only
 -export([new_obj/4,index_id_to_bin/1]).
 
--include_lib("riak_pb/include/riak_pb_kv_codec.hrl").
+-include_lib("riak_protobuf/include/riak_pb_kv_codec.hrl").
 -ifdef(TEST).
 -include_lib("eunit/include/eunit.hrl").
 -endif.
--- a/src/riakc_ts_get_operator.erl
+++ b/src/riakc_ts_get_operator.erl
@@ -24,9 +24,9 @@
 
 -module(riakc_ts_get_operator).
 
--include_lib("riak_pb/include/riak_pb.hrl").
--include_lib("riak_pb/include/riak_ts_ttb.hrl").
--include_lib("riak_pb/include/riak_ts_pb.hrl").
+-include_lib("riak_protobuf/include/riak_pb.hrl").
+-include_lib("riak_protobuf/include/riak_ts_ttb.hrl").
+-include_lib("riak_protobuf/include/riak_ts_pb.hrl").
 
 -export([serialize/3,
          deserialize/1]).
